ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>Add : Otherwise, you can delete the individual resources you created in this tutorial:</p><p> </p><p>These gcloud commands  won't work for uCAIP. The version argument is CAIP only. Replace with uCAIP equivalent.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Okay, I had to delete the last training job cancellation one because it doesn't seem to exist yet for the <code>gcloud ai</code>  commands</p>